app/packages/acter_trigger_auto_complete/lib/src/acter_text_controller.dart

```diff
@@ -25,7 +25,7 @@ class ActerTextController extends TextEditingController {
       final trigger = entry.key;
       final triggerStyle = entry.value;
 
-      final regex = RegExp('\\$trigger\\S+(?: \\S+)*');
+      final regex = RegExp('\\$trigger\\S+(?: \\S+)*(?=\\s)');
```

Comment:
I'd really like some unit tests for the regexp to understand better what to expect from it... and also understand changes made and why...

Reply:
Ahh should've it documented explaining why specifically it's used. Also I will think of some basic tests to include hereafter.